File: week_02/day_3/Homework_revision_fish_river_bear/bear.rb
class Bear

  attr_reader :name

  def initialize(name, type)
    @name = name
    @type = type
    @belly = []
  end

  def food_count()
    return @belly.length()
  end


  def eat_from(river)
    fish = river.remove_fish()
    @belly << fish
  end
end

File: week_06/day_2/MultipleClasses/src/main/java/Bear.java
import java.util.ArrayList;

public class Bear {
    private String name;
    private ArrayList<Salmon> belly;

    public Bear(String name){
        this.belly = new ArrayList<Salmon>();
        this.name = name;
    }

    public int foodCount() {
        return this.belly.size();
    }

    public void eatFishFromRiver(River river) {
        Salmon food = river.removeFish();
        this.belly.add(food);
    }

    public void sleep() {
        this.belly.clear();
    }
}